static/src/javascripts/projects/common/modules/sport/score-board.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import bonzo from 'bonzo';
+import Component from 'common/modules/component';
+import detect from 'lib/detect';
+import $ from 'lib/$';
+
+const getScoreContainerHtml = (context: Object): string => `
+    <div class="score-container">
+        <div class="score__loading ${context.loadingState}">
+            <div class="loading__text">Fetching the scores…</div>
+            <div class="is-updating"></div>
+        </div>
+    </div>
+`;
+
+class ScoreBoard extends Component {
+    constructor(context: Object): void {
+        super(context);
+
+        const scoreContainerHtml = getScoreContainerHtml({
+            loadingState: this.pageType !== 'report'
+                ? ' score__loading--live'
+                : '',
+        });
+
+        this.componentClass = 'match-summary';
+        this.pageType = context.pageType;
```

Comment:
If we're assigning all the properties of `context` to `this`we could replace lines 28-32 with `Object.assign(this, context)`...